src/content/blog/2024-09-29-fighting-the-invisible-Battle-my-story-of-health-struggles.mdx

```diff
@@ -87,15 +90,34 @@ Once I was diagnosed, I realized that going through mental and physical stress w
 
 ## How am I doing now 
 
-I started running, eating healthy, and stopped eating sugar after getting diagnosed. Now, I feel I am physically and mentally in the best shape of my life.
+In the past few months, my mental and physical health has been much better than where I was last year. I am lucky to have Celonis as my employer; the first thing I decided to do after getting diagnosed was take a vacation, and my manager and co-workers fully supported my decision.
+
+I went to Amsterdam with my family and enjoyed exploring its beauty and nearby places. 
+
+I also got an appointment with my diabetes doctor in April 2024. My reports were scary; my doctor thought it was type 1 diabetes and gave me insulin to use for a few days, but in 2 weeks, we got the report, and it was Type 2.
+
+My doctor's first question was, "Are you physically active? Do you go for a walk or run?" 
+
+My answer was "No," and I felt terrible; I bought this day on myself, prioritizing everything apart from my health. I was not eating healthy, either.
+
+My doctor advised me on what I could eat and what I should avoid eating and asked me to start becoming more physically active.
+
+I stopped eating sugar on 15th April 2024 and started walking daily. I also wanted to monitor my health, so I invested in a good fitness watch and bought a Garmin Venu 3. I recommend investing in a good fitness watch.
+
+Running was never in my mind. Initially, I started walking on my treadmill and going for walks outside, but I decided to give it a try and start running over the next few months.
+
+It became a hobby, and I started running three times a week. It became a schedule; no matter what I was doing or where I was, I just left everything and went for a run.
+
```

Comment:
This is great. Nice improvement on adding the details. Did you struggle with giving up sugar?

Reply:
Not really, I just had to prepare myself mentally as I mentioned